back/app/models/user.rb

```diff
@@ -235,13 +236,14 @@ def from_token_request(request)
     where.not(id: project_folder_moderator(*project_folder_ids))
   }
   scope :not_invited, -> { where.not(invite_status: 'pending').or(where(invite_status: nil)) }
-  scope :active, -> { where("registration_completed_at IS NOT NULL AND invite_status is distinct from 'pending'") }
-
-  scope :blocked, lambda {
-    where.not(block_start_at: nil)
-      .and(where(block_start_at: (
-        AppConfiguration.instance.settings('user_blocking', 'duration').days.ago..Time.zone.now)))
+  scope :active, lambda {
+    where("registration_completed_at IS NOT NULL AND invite_status is distinct from 'pending'").not_blocked
+  }
+  scope :active_and_blocked, lambda {
```

Comment:
I remember we were discussing it. But now I think that `active_and_blocked` is a bit confusing (partly because `active` is a bit vague). Maybe `registered` would be better? Or `completed_registration`?